pkg/cprint/cprint.go

```diff
@@ -124,11 +125,11 @@ func PrintDetail(label, value string, verbose bool) {
 
 // CPrinter methods
 func (p *CPrinter) Error(cmd *cobra.Command, err error) {
-	Error(cmd, err, p.verbose)
+	Error(cmd, err, true) //issues/168 introduce always emoji
```

Comment:
If this is defaulted to always be true, why have it as a parameter at all? Is anything actually calling Error directly? Let's remove dead code if there is dead code.

Same comment with the other two below this.

IMO, it's weird to have some of these (Error, Info and Success) ignore the verbose flag and behave differently than the other ones here (YellowPrint and Warning). I think it'd be better to still have the concept of verbose/not.

https://github.com/Hyphen/hx/issues/168 is specifically about the output of the hx push command. I think it's a little bit of doing too much to change the behavior of all Error, Info and Success messages. Just my 2c. Maybe clarify with the card author if they want other commands to also include that emoji?

Reply:
@kylekampy Yep I agree, but I had a card to remove the emojis and colors in the past,  but now, i got one to add them just for this ones? @osbornm, @jaredwray , do we want other command to have emojist too?

Reply:
Why do we not want to include emoji? We need to improve how things look and make it more magical.